transport/src/test/java/io/netty/channel/embedded/EmbeddedChannelTest.java

```diff
@@ -626,7 +628,7 @@ public void channelRegistered(ChannelHandlerContext ctx) {
             }
         });
 
-        final EmbeddedEventLoop embeddedEventLoop = new EmbeddedEventLoop();
+        final EmbeddedEventLoop embeddedEventLoop = new EmbeddedEventLoop(new EmbeddedEventLoop.FreezableTicker());
```

Comment:
Is this change needed?

Reply:
EmbeddedEventLoop is package private so I think it's fine